bin/apibuilder

```diff
@@ -112,21 +112,12 @@ end
 # that corresponds to an api version that was uploaded to ApiBuilder. Looks
 # backwards a max of 100 commits.
 def head_of_latest_uploaded_version(org, application, client)
-  found = nil
-  0.upto(99).each do |i|
-    version = ApibuilderCli::Git.generate_version(i)
-    begin
-      apibuilder_version = client.versions.get_by_application_key_and_version(org, application, version)
-      found = i
-      break
-    rescue Io::Apibuilder::Api::V0::HttpClient::ServerError => e
-      # Continue looking when the version is not found; fail for other exceptions.
-      if e.code != 404
-        raise e
-      end
-    end
-  end
-  found
+  versions = client.versions.get_by_application_key(org, application, limit: 100).map { |v| v.version }
```

Comment:
the api says:
```
The latest version is the first result returned.
```

https://app.apibuilder.io/apicollective/apibuilder-api/latest

so I think we can just limit 1

Reply:
we are actually hitting this endpoint: https://app.apibuilder.io/apicollective/apibuilder-api/latest#resource-version

and I don't think we can do limit=1 because what if there are new versions in apibuilder that you don't have locally?

Reply:
ack